### DISCUSSION TITLE
I have created one form builder. But I can understand How I can make accessible to the world.

### DISCUSSION THREAD

**Discussion by swrno** [GitHub-Octernships/Shuffle-Frontend-Developer-Software-Engineering#8] [Q&A]
### **I have created the form Interface**
![Screenshot (18)](https://github.com/GitHub-Octernships/Shuffle-Frontend-Developer-Software-Engineering/assets/94161758/67dfd239-0148-4037-90cb-e64c7b3b200f)

**1. How can I add the data to database ?
2. How can I make the form available to the world?
3. How to collect the responses of form?**

@ashwinexe , @jaygohil-cyber can you please guide me

(You can pin any document which explains how to use Database in Shuffle)

**Comment by jaygohil-cyber** [comment]
For 2 & 3, that's a part of the assignment!

For 1, here you go: https://github.com/Shuffle/Shuffle/blob/main/.github/install-guide.md
